ios/Cartfile

```diff
@@ -1,5 +1,5 @@
 github "marmelroy/Zip"
-github "DaveWoodCom/XCGLogger" ~> 6.1.0
+github "keymanapp/dependency-XCGLogger" "master"
```

Comment:
Do we want a specific tag here? The latest looks to be 7.0.1 from Feb 2020

Reply:
I think we can stick with 'master' given we control the fork

Reply:
You use 'master' here but 'head' in oem/firstvoices/ios/Cartfile?

```
github "keymanapp/dependency-XCGLogger" "master"
```

vs 

```
github "keymanapp/dependency-XCGLogger" "head"
```

Can we use the same in both places please?